### PR TITLE
maps inspection fix

### PR DESCRIPTION
**Pull request by sqreek**
Cleanup from Intellij inspection.
Mostly name renaming, no logic changes.
If tests pass no Qa needed.

@mixth-sense @macbre

### CODE REVIEW

File: extensions/wikia/WikiaMaps/css/WikiaMaps.wikiamobile.scss [hunk 1]
```diff
@@ -92,7 +92,7 @@ $map-ad-height: 70px;
 
 		a {
 			border-radius: 5px;
-			display: block-inline;
+			display: inline-block;
```
Comment (macbre) [inline extensions/wikia/WikiaMaps/css/WikiaMaps.wikiamobile.scss:95]: I'd double check this one. Browser will likely use `display` property from another selector based on CSS inheritance. So yes, by fixing we can brake things :)

File: extensions/wikia/WikiaMaps/tests/WikiaMapsPoiControllerTest.php [hunk 2]
```diff
@@ -57,6 +60,7 @@ public function testDeletePoi_throws_permission_error_when_anon() {
 	public function testDeletePoi_throws_permission_error_when_blocked() {
 		$userMock = $this->getUserMock( self::USER_TYPE_BLOCKED );
 
+		/** @var WikiaMapsPoiController $controllerMock */
 		$controllerMock = $this->getWikiaMapsPoiControllerMock();
```
Comment (macbre) [inline extensions/wikia/WikiaMaps/tests/WikiaMapsPoiControllerTest.php:60]: Can we instead add PHPdoc to `getWikiaMapsPoiControllerMock` method?

Comment (sqreek) [inline extensions/wikia/WikiaMaps/tests/WikiaMapsPoiControllerTest.php:60]: yep. Good call

**Comment by macbre** [comment]
:+1:

**Pull request closed without merging**